core/src/actions/account.ts

```diff
@@ -1,15 +1,14 @@
+import { ParamsFrom } from "actionhero";
 import { AuthenticatedAction } from "../classes/actions/authenticatedAction";
 import { TeamMember } from "../models/TeamMember";
+import { ActionPermission } from "../models/Permission";
 
 export class AccountView extends AuthenticatedAction {
-  constructor() {
-    super();
-    this.name = "account:view";
-    this.description = "view your account details";
-    this.permission = { topic: "*", mode: "read" };
-    this.outputExample = {};
-    this.inputs = {};
-  }
+  name = "account:view";
+  description = "view your account details";
+  permission: ActionPermission = { topic: "*", mode: "read" };
+  outputExample = {};
+  inputs = {};
```

Comment:
**Main change #1**

Action and Task options are statically defined rather than as part of a constructor.  This is how you can stongly type them